### ISSUE TITLE
Add keyboard controls for the 2D direction vector control.

### ISSUE THREAD

**Issue by jasonwebb** [jasonwebb/supertunnel#3]
This 2D control can be operated by mouse and touch but not by keyboard, which is a violation of [WCAG 2.1.1](https://www.w3.org/WAI/WCAG21/Understanding/keyboard.html). Adding a toggleable, arrow key controls, and some instructions should fix that.

- [x] Turn the drag handle into a `<button>`, since it is actionable.
- [x] Add `aria-pressed` to the drag handle button to indicate when it is actively being dragged.
  - [x] Dynamically set to `true` when dragging, then `false` when not.
  - [x] Set it to `false` automatically when the drag handle button is blurred.
- [x] Provide instructions explaining how to activate and use the keyboard dragging functionality. Ideally these are visible for all users.
